### ISSUE TITLE
Alternative to Verbose CSS variables

### ISSUE THREAD

**Issue by d3dc** [sveltekit/ui#7]
I guess the reason for the verbose naming is to avoid collision with the user's defined values. If I have a one-off Card or similar, its a lot of typing `--Card-` in the style attribute.

It might be convenient to expose Component-level CSS variables as props:

```svelte
<script>
  export let borderRadius
</script>

<div style="--Card-border-radius: {borderRadius}">
</div>
```

**Comment by rob-balfre** [comment]
@d3dc yeah jury is still out on how we are gonna handle CSS vars. Will keep this open...
